docker/entrypoint.sh

```diff
@@ -1,2 +1,3 @@
 #!/bin/sh
+python -m bothub-nlp import_langs -e=${EXTRA_MODELS_DIR} ${SUPPORTED_LANGUAGES}
```

Comment:
Involve the variable expansion ${} with double quotes. Like that: "${SUPPORTED_LANGUAGES}"